tensorflow/core/transforms/remapper/pass.cc

```diff
@@ -84,14 +88,177 @@ class MatchMulSigmoid : public RewritePattern {
   OperationName sigmoid_name_;
 };
 
+static FailureOr<TFOp> CreateContractionWithBiasAddOp(PatternRewriter &rewriter,
+                                                      OpPropertyHelper &helper,
+                                                      Operation *contraction_op,
+                                                      Operation *bias_add_op) {
+  // Get all operands for fused op
+  Value input = contraction_op->getOperand(0);
+  Value filter = contraction_op->getOperand(1);
+  Value bias = bias_add_op->getOperand(1);
+
+  SmallVector<Value> operands;
+  operands.push_back(input);
+  operands.push_back(filter);
+  operands.push_back(bias);
```

Comment:
You may need to also propagate the control operands if you are replacing this op